core-web/libs/portlets/dot-content-drive/portlet/src/lib/components/dot-content-drive-toolbar/dot-content-drive-toolbar.component.ts

```diff
@@ -12,9 +14,12 @@ import { DotContentDriveTreeTogglerComponent } from './components/dot-content-dr
         ToolbarModule,
         ButtonModule,
         DotContentDriveTreeTogglerComponent,
+        DotContentDriveBaseTypeSelectorComponent,
+        DotContentDriveContentTypeFieldComponent,
```

Comment:
Wonder if this two should be common selectors that we can have in our `ui` folder to reuse... just asking.

Reply:
Don't we have a content type selector already?

Reply:
we have it, but it is a single selector, making it multi select will require changes across the app, which is not the scope of this issue

Reply:
I think this kind of selector is only used on Content Types portlet